src/adapters/superagent.js

```diff
@@ -1,30 +1,13 @@
 import superagent from 'superagent';
 import * as httpMethods from '../constants/http-methods';
 
-export const createRequest = (url, method) => {
-    switch (method) {
-        case httpMethods.GET:
-            return superagent.get(url);
-        case httpMethods.POST:
-            return superagent.post(url);
-        case httpMethods.PUT:
-            return superagent.put(url);
-        case httpMethods.PATCH:
-            return superagent.patch(url);
-        case httpMethods.DELETE:
-            return superagent.del(url);
-        default:
-            throw new Error(`Unsupported HTTP method: ${method}`);
-    }
-};
-
 const superagentNetworkAdapter = (url, method, { body, headers, credentials } = {}) => {
-    const request = createRequest(url, method);
-
-    if (body) {
-        request.send(body);
+    if (!(method in httpMethods)) {
```

Comment:
Now that I think of it, this is completely wrong as it is checking against the names of the constants, rather than their values. It works because they're the same.